sdk/keyvault/Azure.Security.KeyVault.Keys/samples/Sample6_WrapUnwrap.md

```diff
@@ -0,0 +1,73 @@
+# Wrapping and unwrap a key
+
+This sample demonstrates how to wrap and unwrap a symmetric key with an RSA key.
+To get started, you'll need a URI to an Azure Key Vault. See the [README](../README.md) for links and instructions.
+
+## Creating a KeyClient
+
+To create a new `KeyClient` to create, get, update, or delete keys, you need the endpoint to a Key Vault and credentials.
+You can use the `DefaultAzureCredential` to try a number of common authentication methods optimized for both running as a service and development.
+
+In the sample below, you can set `keyVaultUrl` based on an environment variable, configuration setting, or any way that works for your application.
+
+```C# Snippet:KeysSample6KeyClient
+var keyClient = new KeyClient(new Uri(keyVaultUrl), new DefaultAzureCredential());
+```
+
+## Creating a key
+
+First create an RSA key which will be used to wrap and unwrap another key
```

Comment:
First, - last comment I'll make on this, please check doc for First, Then, Next,

Reply:
Replaced all matching regex `^(First|Next|Then)(?!,)\s`.